buildroot-external/patches/homematic/0014-WebUI-ProgramListSort.patch

```diff
@@ -226,47 +223,63 @@
              }
            }
          }
-@@ -128,6 +293,7 @@
+@@ -128,12 +290,13 @@
      %>
    </tbody>
  </table>
 +</div>
  
  <script type="text/javascript">
    DeleteProgram = function(id)
-@@ -233,6 +399,7 @@
+   {
+     var _id_ = id;
+-    //new YesNoDialog("Sicherheitsabfrage", "M�chten Sie das Programm wirklich l�schen?", function(result) {
++    //new YesNoDialog("Sicherheitsabfrage", "Möchten Sie das Programm wirklich löschen?", function(result) {

```

Comment:
Auch hier hat sich anscheinend eine UTF8 Kodierung eingeschlichen und es wird daher eine Änderung der Zeile angezeigt. Sowas sollte vermieden werden.

Reply:
Auch an dieser Zeile habe ich selber nicht verändert. Seltsamerweise sind bei mir lokal sowohl "programlist.htm" wie auch "programlist.htm.orig" nach wie vor korrekt. Wird hier durch die "create_patches.sh" eventuell etwas verbogen?

Reply:
Nein, schau dir doch mal das an:

https://github.com/jens-maus/RaspberryMatic/pull/154/files#diff-5b4a267a68a4f9afc2d2e4efab36cbaeL6

D.h. du hast da definitiv etwas verändert in der Zeile und eine Leerzeile rausgemacht. und das das in den .patch files dann so landet ist nur logisch.